sdk/maps/maps-weather-rest/test/public/mapsWeather.spec.ts

```diff
@@ -0,0 +1,170 @@
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { env, Recorder } from "@azure-tools/test-recorder";
+import { Context } from "mocha";
+import MapsWeather, { MapsWeatherClient } from "../../src";
+import { AzureKeyCredential } from "@azure/core-auth";
+import { assert } from "chai";
+
+describe("MapsWeatherClient Tests", function () {
+  let recorder: Recorder;
+  let client: MapsWeatherClient;
+
+  beforeEach(async function (this: Context) {
+    recorder = new Recorder(this.currentTest);
+    await recorder.start({
+      envSetupForPlayback: {
+        MAPS_RESOURCE_CLIENT_ID: "azure_maps_client_id",
+      },
+    });
+    client = MapsWeather(new AzureKeyCredential(env["AZURE_SUBSCRIPTION_KEY"] as string));
```

Comment:
Please follow other SDK, using the `createClient()` to avoid using key credential in the test.